SpeechService/Translations.cs

```diff
@@ -678,6 +678,7 @@ public static string Humanize(decimal? value)
             int nextDigit;
             string order;
             int digits = (int)Math.Log10((double)value);
+            bool morePreciseDecimalValues = digits < 10;
```

Comment:
Why this criterion, and why hard-coded? 
Also, why not just compare `value < 1e10` which would be much quicker (single machine op vs log10 call)?

Reply:
I admit, this one's a bit of an arbitrary number. The thought is that once we're dealing with numbers larger than 10 billion, I don't really care to hear about half a billion, half a trillion, etc. and am looking for a way to reduce the cognitive burden.